functions/slack/main.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018, Google, LLC.
+# Copyright 2022 Google LLC.
```

Comment:
```suggestion
# Copyright 2018 Google LLC.
```
Please do not update the years! It should reflect when the file was first introduced.